.github/workflows/release.yaml

```diff
@@ -0,0 +1,242 @@
+name: Concordium desktop wallet release
+
+on:
+  push:
+    tags:
+      - '*/*.*.*'
```

Comment:
```suggestion
      - '*/*.*.*-stagenet'
      - '*/*.*.*-testnet'
      - '*/*.*.*-mainnet'
```

Reply:
This is obviously to ensure the right environments are used, but also to help the releaser knowing what tag names are supposed to be